README.md

```diff
@@ -52,12 +84,44 @@ in [GitHub Pages](https://tbd54566975.github.io/web5-kt/docs/htmlMultiModule/cre
 
 # Development
 
+## Prerequisites
+
+Install java version 11. If you're installing a higher version, it must be compatible with Gradle 8.2.
+
+If you want to have multiple version of Java installed in your machine, we recommend using [jenv](https://www.jenv.be/).
+
+> [!NOTE]: Restart your shell after installation.
+
+## Build
+
 To build and run test just run:
 
 ```bash
 ./gradlew build --console=rich
 ```
 
+## Releasing
+
+In order to release to Central Repository, simply cut a new release tag in the Github UI. There is a configured [Github
```

Comment:
Does this allow for bumping individual packages, or will this bump everything?

Reply:
Does this action release to both jitpack and maven central?

Reply:
> Does this allow for bumping individual packages, or will this bump everything?

It does not allow for bumping individual packages. We'll have to bump everything.

> Does this action release to both jitpack and maven central?

Jitpack works differently than maven. You don't need to "release". Instead, it builds a JAR on the fly when you ask for a specific version. You can refer to which JAR by using a PR, tag, or commit.